payload/builder.go

```diff
@@ -11,14 +11,16 @@ type Payload struct {
 }
 
 type aps struct {
-	Alert            interface{} `json:"alert,omitempty"`
-	Badge            interface{} `json:"badge,omitempty"`
-	Category         string      `json:"category,omitempty"`
-	ContentAvailable int         `json:"content-available,omitempty"`
-	MutableContent   int         `json:"mutable-content,omitempty"`
-	Sound            interface{} `json:"sound,omitempty"`
-	ThreadID         string      `json:"thread-id,omitempty"`
-	URLArgs          []string    `json:"url-args,omitempty"`
+	Alert             interface{} `json:"alert,omitempty"`
+	Badge             interface{} `json:"badge,omitempty"`
+	Category          string      `json:"category,omitempty"`
+	ContentAvailable  int         `json:"content-available,omitempty"`
+	MutableContent    int         `json:"mutable-content,omitempty"`
+	Sound             interface{} `json:"sound,omitempty"`
+	ThreadID          string      `json:"thread-id,omitempty"`
+	URLArgs           []string    `json:"url-args,omitempty"`
+	InterruptionLevel string      `json:"interruption-level,omitempty"`
```

Comment:
The keys in the original struct were alphabetically ordered. We should probably continue doing that.

Reply:
Updated as per suggestion.